package.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JSON is not sorted](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4765301)